globalSetup.ts

```diff
@@ -2,6 +2,8 @@ import { GenericContainer, Wait } from "testcontainers"
 
 export default async function setup() {
   await new GenericContainer("budibase/couchdb")
+    .withName("budibase-test-couchdb")
```

Comment:
Do we need the name? What will happen if you are running 2 test consoles locally in parallel?

Reply:
We don't need it, no, we could have as many containers running in parallel as we want. I did test this and it works fine. But reusing the containers gives a non-trivial performance benefit.